app/controllers/api/v1/users_controller.rb

```diff
@@ -14,9 +14,9 @@ def show
   def index
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/MethodLength
**Severity**: INFO
**File**: app/controllers/api/v1/users_controller.rb L14

Method has too many lines. [16/10]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*